docs/cli.rst

```diff
@@ -88,7 +88,8 @@ Commands and flags
       --batch-name BATCH_NAME
                             Specify a value for a `batchname` column to append to
                             each row.
-
+      --officeids OFFICEIDS Specify officeids to parse
```

Comment:
I'd put this higher up in the order, nearer to similar settings like `--results-level` and `--raceids`. Think of the UX from the perspective of a new dev learning the settings from `elex --help`.